tests/python/gaia-ui-tests/gaiatest/apps/contacts/app.py

```diff
@@ -26,6 +26,8 @@ class Contacts(Base):
     _no_contacts_message_locator = (By.CSS_SELECTOR, '*[data-l10n-id="no-contacts"]')
     _group_container_selector = "#groups-container"
     _contact_locator = (By.CSS_SELECTOR, 'li[data-uuid]:not([data-group="ice"])')
+    _ice_list_icon_locator = (By.ID, 'contact-list-ice')
+    _ice_contact_list_locator = (By.ID, 'p[data-l10n-id="ICEContactsGroup"]')
```

Comment:
`By.ID` shouldn't work, as it's a regular CSS selector. So as the test doesn't fail, I guessed that this selector is unused.